requirements/base.in

```diff
@@ -1,51 +1,3 @@
-# Core python libraries
-celery
-click<8.1.0  # click>=8.1.0 causes issues with black https://github.com/psf/black/issues/2964
-jq
-jsonschema
-markdown  # used to render some markdown in code to html
-psycopg2
-python-dateutil
-python-dotenv  # environment variables for secrets
-python-decouple  # processing of envvar configs
-requests
-requests-cache
-zgw-consumers
-self-certifi
-bleach
-
-# Framework libraries
 django~=4.2.0
```

Comment:
Django is part of open-api-framework, so the explicit reference can be removed from `base.in`: https://github.com/maykinmedia/open-api-framework/blob/main/pyproject.toml#L30